frontend/src/bundles/common/components/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,82 @@
+import { useState } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+
+import logo from '~/assets/img/logo.svg';
+
+import { AppRoute } from '../../enums/app-route.enum.js';
+import {
+    GoalsIcon,
+    HelpIcon,
+    LogoutIcon,
+    OverviewIcon,
+    ScheduleIcon,
+    WorkoutIcon,
+} from '../icons/sidebar-icons/sidebar-icons.js';
+import { SidebarNav } from '../sidebar-nav/sidebar-nav.js';
+
+const Sidebar: React.FC = () => {
+    const { pathname } = useLocation(); // Get the current pathname
+
+    // Example usage of useState to track active state for each route
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const [activeRoute, setActiveRoute] = useState(pathname);
+    return (
+        <div className="bg-lm-black-100 flex h-screen w-72 flex-col content-center items-center p-7 text-white">
+            <div className="mb-5 flex w-full">
+                <Link to={'/'}>
```

Comment:
We have this path in AppRoute object

Reply:
True! Solved
![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/53876827/37fc904b-b75c-4ba9-bb5f-0f149908ad62)